addons/mail/static/src/models/composer/composer.js

```diff
@@ -102,6 +102,19 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * Focus this composer and remove focus from all others.
```

Comment:
please explain why (focus is a global concern and it makes no sense to have multiple composers focused at the same time)